src/main/java/com/patsi/service/SocialMediaService.java

```diff
@@ -18,26 +27,77 @@ public class SocialMediaService {
     @Autowired
     private SocialMediaRepository socialMediaRepository;
 
-    //Edit User Profile Picture
-    public void editSocialMediaAccount(SocialMediaUser user) {
+    @Value("${spring.web.resources.static-locations[5]}")
+    private String IMAGE_PATH_ProfilePicture;
+    @Value("${spring.web.resources.static-locations[6]}")
+    private String IMAGE_PATH_BannerPicture;
+
+    public UUID getUserUid() {
+        UUID uid = UUID.randomUUID();
+        return uid;
+    }
+
+    public SocialMediaUser createSocialMediaAccount( SocialMediaUser user) {
+        log.info("In service: createSocialMediaAccount");
+        SocialMediaUser.builder()
+            //Todo: Get uid from Login Profile
+            .uid(user.getUid())
+            .displayName(user.getDisplayName())
+            .biography(user.getBiography())
+            .accountStatus(AccountStatus.Active)
+            .accountType(user.getAccountType())
+            .build();
         socialMediaRepository.save(user);
+        user.setUid(null);
+        return user;
+    }
+
+    public void changeProfilePicture(String profilePictureID, byte[] profilePicture)
+        throws IOException {
+        log.info("In Service: changeProfilePicture");
+            File f = FileHelper.newFile(IMAGE_PATH_ProfilePicture + profilePictureID +".jpg");
+            try(FileOutputStream outputStream = FileHelper.newFileOutputStream(f)){
+                outputStream.write(profilePicture);
+            }
+    }
+
+    public SocialMediaUser changeBannerPicture(String userName, byte[] bannerPicture) throws IOException {
+        log.info("In Service: changeBannerPicture");
+        SocialMediaUser user = socialMediaRepository.findByUserName(userName).orElse(null);
+        if(user != null){
+            File f = FileHelper.newFile(IMAGE_PATH_BannerPicture + user.getUserName() +".jpg");
```

Comment:
same here for extracting to file util.
`think you can combine the whole writing file logic into one single method in FileHelper, which accept a file path + name and the byte[].`